### ISSUE TITLE
Wrong AC-2 enhancement listed in the component examples

### ISSUE THREAD

**Issue by iMichaela** [usnistgov/oscal-content#20]
# Describe the bug

The component examples are citing the wrong enhancement of the AC-2 control . The AC-2.3 is addressing the inactive accounts not AC-2.2

# What files is the bug affecting?
https://github.com/usnistgov/OSCAL/blob/master/content/components/json/example-component-min.json
https://github.com/usnistgov/OSCAL/blob/master/content/components/json/example-component.json 

# When does this occur?

When the example was created

# Expected behavior (i.e. solution)

Replace any reference to AC-2.2 with AC-2.3

# Other Comments

none.

**Comment by iMichaela** [comment]
## 12/034/2020

This issue was not addressed by the PR #32 .

The component example indicates that it implements the control enhancement AC-2(2) from Rev4 which reads:

```
(2)	ACCOUNT MANAGEMENT | REMOVAL OF TEMPORARY / EMERGENCY ACCOUNTS
The information system automatically [Selection: removes; disables] temporary and emergency accounts after [Assignment: organization-defined time period for each type of account].
Supplemental Guidance:  This control enhancement requires the removal of both temporary and emergency accounts automatically after a predefined period of time has elapsed, rather than at the convenience of the systems administrator.
```

But the OSCAL component example reproduced below addresses inactive accounts: 

```
"control-implementations": [
          {
            "uuid": "cfcdd674-8595-4f98-a9d1-3ac70825c49f",
            "source": "../../../content/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.json",
            "description": "This is a partial implementation of the SP 800-53 rev4 catalog, focusing on the control enhancement AC-2 (2).",
            "implemented-requirements": [
              {
                "uuid": "d1016df0-9b5c-4839-86cd-f9c1d113077b",
                "description": "Inactive accounts are automatically disabled based on the duration specified by the duration parameter. Disabled accounts are expected to be reviewed and removed when appropriate.",
                "control-id": "ac-2.2"
              }
            ]
          },
```

 AC-2(3) deals with inactive accounts, not not AC-2(2).  

AC-2(3) reads in 800-53 r4:

```
(3)	ACCOUNT MANAGEMENT | DISABLE INACTIVE ACCOUNTS 
The information system automatically disables inactive accounts after [Assignment: organization-defined time period].
```

**Comment by david-waltermire-nist** [comment]
True. PR #32 focused on changes to the SP 800-53 content. Related content has not been updated yet. This issue is still open because this still needs to be fixed.
